jobs/build/update-base-images/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+#➜  ~ docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/openjdk18-rhel7:latest id
+#uid=185(jboss) gid=185(jboss) groups=185(jboss)
+#➜  ~ docker run -it  brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/elasticsearch:latest id
+#uid=1000(elasticsearch) gid=1000(elasticsearch) groups=1000(elasticsearch)
+#➜  ~  docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/ansible-runner:latest  id
+#uid=0(root) gid=0(root) groups=0(root)
+# `rhel7` is root 0`
+# `nodejs6` is 1001`
+
+TAG="rhaos-4.0-rhel-7"
+URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h=${TAG}"
+USER_USERNAME="--user=ocp-build"
+
+build_common() {
+    cd openshift-enterprise-base
+    echo "$1" > additional-tags
+    echo """FROM $2
+
+    USER root
+    RUN yum update -y && yum clean all
+    USER $3
+
+    LABEL \\
+            com.redhat.component=\"openshift-enterprise-base-container\" \\
+            name=\"openshift/ose-base\" \\
+            version=\"v4.0\" \\
+            release=\"$(date +%Y%m%d%H%M)\"
+    """ > Dockerfile
+    git commit -am "updated $1 container"
+    git push
+    rhpkg  ${USER_USERNAME} container-build --repo-url ${URL}
+}
+
+case "$1" in
+        ansible-runner)
+            rm -rf openshift-enterprise-base
+            rhpkg ${USER_USERNAME} clone --branch ${TAG} containers/openshift-enterprise-base
```

Comment:
these first two lines are the same each time. why not put them in `build_common`?

BTW I changed my mind on reusing the clone between builds. It's not that much to clone, and it would prevent race conditions where something else (maybe another instance of this job) updates the dist-git during builds. Given how long builds can take, that's not even unlikely. So, delete and re-create is fine. (Although git pull would probably work too.)